parlai/agents/transformer/transformer.py

```diff
@@ -91,6 +91,13 @@ def add_common_cmdline_args(argparser):
         help='Share word embeddings table for candidate and context'
         'in the memory network',
     )
+    argparser.add_argument(
+        '--get-pos-embs',
+        type='bool',
+        default=False,
+        help='If true, return the position embeddings from the encoder '
```

Comment:
This feels like it shouldn't be a user option, or at least could be rolled into the reduction_type option?

Reply:
i think this might be the way to go; at first I had just turned it on by default but figured people would want to have a way to turn it off. but changing to work with reduction type of, e.g., `none_with_pos` might be better